modules/nw-networkpolicy-about.adoc

```diff
@@ -16,26 +16,26 @@ means that egress policy types, IPBlock, and combining `podSelector` and
 `namespaceSelector` are not available in {product-title}.
 ====
 
-In a cluster network isolation is controlled entirely by
+In a cluster, network isolation is controlled entirely by
 link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network/network-policy.md[`NetworkPolicy`
-objects]. By default, all pods in a project are accessible from other pods and
-network endpoints. To isolate one or more pods in a project, you can create
+objects]. By default, all Pods in a project are accessible from other Pods and
+network endpoints. To isolate one or more Pods in a project, you can create
 `NetworkPolicy` objects in that project to indicate the allowed incoming
 connections. Project administrators can create and delete `NetworkPolicy`
 objects within their own project.
```

Comment:
Is ownership a factor here? Why do we mention Project admins? Are cluster admins handled differently?